src/main/java/com/github/princesslana/eriscasper/rest/webhook/ExecuteGithubWebhookRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.github.princesslana.eriscasper.rest.webhook;
+
+import com.github.princesslana.eriscasper.util.QueryStringBuilder;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/webhook#execute-githubcompatible-webhook-querystring-params">
+ *     https://discordapp.com/developers/docs/resources/webhook#execute-githubcompatible-webhook-querystring-params</a>
+ */
+@Value.Immutable
+public interface ExecuteGithubWebhookRequest {
+  Optional<Boolean> isWait();
+
+  default String toQueryString() {
+    return new QueryStringBuilder().addBool("wait", isWait()).build();
```

Comment:
I don't see a `addBool` method added to query string (you may have missed including that change here), but let's not abbreviate inconsistently. I'd prefer `addBoolean`

Reply:
I need to add that into the branch, forgot to add the utility upgrades to this branch.